modules/cnf-installing-clusters.adoc

```diff
@@ -0,0 +1,150 @@
+// Epic CNF-9657 (TELCODOCS-1540) (4.16)
+// * edge_computing/cnf-understanding-siteconfig-operator.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="cnf-installing-clusters_{context}"]
+= Installing {sno} clusters with the {sco}
+
+Install your clusters with the {sco} by using the default cluster templates.
+
+.Prerequisites
+
+* You have logged in to the hub cluster as a user with cluster-admin privileges.
+* If using {ztp}, you have configured your {ztp} environment.
+* You have the default cluster templates.
+//Anything else?
+
+.Procedure
+
+. Create your cluster and node templates for the Image-based Install Operator by running the following commands:
++
+--
+[source,terminal]
+----
+$ oc apply -f ibi-cluster-templates-v1.yaml
+----
+
+[source,terminal]
+----
+$ oc apply -f ibi-node-templates-v1.yaml
+----
+--
+
+. Create the `ClusterInstance` CR, named for example `clusterinstance-ibi.yaml`, and reference your templates:
++
+[source,yaml]
+----
+spec:
+  [...]
+  nodes:
+    templateRefs: <1>
+      - name: ibi-node-templates-v1.yaml
+        namespace: siteconfig-system
+    [...]
+  templateRefs: <2>
+    - name: ibi-cluster-templates-v1.yaml
+      namespace: siteconfig-system
+----
+<1> Reference node-level templates under the `spec.nodes.templateRefs` field.
+<2> Reference cluster-level templates under the `spec.templateRefs` field.
+
+. Create the `ClusterInstance` CR by running the following command:
++
+--
+[source,terminal]
+----
+$ oc apply -f clusterinstance-ibi.yaml
+----
+
+After creating the CR, the {sco} starts reconciling the `ClusterInstance` CR, then validates and renders the installation manifests.
+The {sco} continues to monitor for changes in the `ClusterDeployment` CRs to update the cluster installation progress of the corresponding `ClusterInstance` CR.
+--
+
+.Verification
+
+. Monitor the process by running the following command:
++
+--
+[source,terminal]
+----
+$ oc get siteconfig -o json
+----
+//please check command
+
+.Example output for successful validation
+[source,terminal]
+----
+     message: Validation succeeded
+     reason: Completed
+     status: "True"
+     type: SiteConfigValidated
+----
+
+.Example output for successful rendering
+[source,terminal]
+----
+     message: Rendered templates validation succeeded
+     reason: Completed
+     status: "True"
+     type: RenderedTemplates
+----
+
+.Example output for successful dry run
+[source,terminal]
+----
+     message: Rendered templates validation succeeded
+     reason: Completed
+     status: "True"
+     type: RenderedTemplatesValidated
+----
+
+.Example output for successful manifest generation
+[source,terminal]
+----
+     message: Created site config manifests, provisioning cluster
+     reason: InProgress
+     status: "True"
+     type: Provisioned
+----
+--
+
+. Check the manifests that {sco} rendered by running the following command:
++
+--
+[source,terminal]
+----
+$ oc get siteconfig -o json
+----
```

Comment:
I think we can combine this with the above and showcase the `deploymentConditions` with the `manifestsRendered` as well:
```yaml
  deploymentConditions:
    - lastProbeTime: "2024-07-08T21:06:45Z"
      lastTransitionTime: "2024-07-08T21:05:42Z"
      message: The cluster is not ready to begin the installation
      reason: ClusterNotReady
      status: "False"
      type: ClusterInstallRequirementsMet
    - lastProbeTime: "2024-07-08T21:06:45Z"
      lastTransitionTime: "2024-07-08T21:05:42Z"
      message: The installation has not yet started
      reason: InstallationNotStarted
      status: "False"
      type: ClusterInstallCompleted
    - lastProbeTime: "2024-07-08T21:06:45Z"
      lastTransitionTime: "2024-07-08T21:05:42Z"
      message: The installation has not failed
      reason: InstallationNotFailed
      status: "False"
      type: ClusterInstallFailed
    - lastProbeTime: "2024-07-08T21:06:45Z"
      lastTransitionTime: "2024-07-08T21:05:42Z"
      message: The installation is waiting to start or in progress
      reason: InstallationNotStopped
      status: "False"
      type: ClusterInstallStopped
  manifestsRendered:
    - apiGroup: extensions.hive.openshift.io/v1beta1
      kind: AgentClusterInstall
      lastAppliedTime: "2024-07-08T21:06:45Z"
      name: cnfdg6
      namespace: cnfdg6
      status: rendered
      syncWave: 1
    - apiGroup: metal3.io/v1alpha1
      kind: BareMetalHost
      lastAppliedTime: "2024-07-08T21:06:45Z"
      name: cnfdg6.ptp.eng.rdu2.dc.redhat.com
      namespace: cnfdg6
      status: rendered
      syncWave: 1
    - apiGroup: hive.openshift.io/v1
      kind: ClusterDeployment
      lastAppliedTime: "2024-07-08T21:06:45Z"
      name: cnfdg6
      namespace: cnfdg6
      status: rendered
      syncWave: 1
    - apiGroup: agent-install.openshift.io/v1beta1
      kind: InfraEnv
      lastAppliedTime: "2024-07-08T21:06:45Z"
      name: cnfdg6
      namespace: cnfdg6
      status: rendered
      syncWave: 1
    - apiGroup: agent-install.openshift.io/v1beta1
      kind: NMStateConfig
      lastAppliedTime: "2024-07-08T21:06:45Z"
      name: cnfdg6.ptp.eng.rdu2.dc.redhat.com
      namespace: cnfdg6
      status: rendered
      syncWave: 1
    - apiGroup: agent.open-cluster-management.io/v1
      kind: KlusterletAddonConfig
      lastAppliedTime: "2024-07-08T21:06:45Z"
      name: cnfdg6
      namespace: cnfdg6
      status: rendered
      syncWave: 2
    - apiGroup: cluster.open-cluster-management.io/v1
      kind: ManagedCluster
      lastAppliedTime: "2024-07-08T21:06:45Z"
      name: cnfdg6
      status: rendered
      syncWave: 2
```

Reply:
same as for status conditions